src/core/grade_notifier.py

```diff
@@ -211,35 +212,42 @@ def start_notifier():
 
 
 def check_user_exists(username):
+    stored_username = custom_hash(username)
```

Comment:
I figured that it was a better idea to only use the hashed results in the function given and have each function do its own hashing because otherwise there wouldn't be an across the board standard for which functions take in the username and which take in the stored_username. It also allows things to work the way they were originally intended to and allows me to not edit any test cases I think.

Reply:
Makes sense to me. Thumbs up